### PR TITLE
Convert ha-long-lived-access-tokens-card

### LINKED ISSUE #6888: Creating Long-Lived Access Tokens from the profile page doesn't display the token
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

I can create a LLAT from the profile page in Lovelace and give it a name, but the value of the token is never displayed.  The token *is* created and added to the list with the proper name, and I can dig around in `config/auth` to find the actual token value.

## Expected behavior

A popup or similar should appear during token creation to show the token value.

## Steps to reproduce

Go into the profile page in Lovelace and click "Create Token" at the bottom.  Provide a name, click OK, and note that the new token is added to the list, but the value is never shown.  It does appear that the token is created properly because I can find it in `config/auth`.

## Environment

- Home Assistant release with the issue: 0.115.0b3
- Last working Home Assistant release (if known): Unknown.  It did work at some point.
- Browser and browser version: Chrome 85
- Operating system: Windows/MacOS

### PR DESCRIPTION
**Pull request by ludeeus**
## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #6888
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/profile/ha-long-lived-access-tokens-card.ts [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+  showConfirmationDialog,
+  showPromptDialog,
+} from "../../dialogs/generic/show-dialog-box";
+import { haStyle } from "../../resources/styles";
+import "../../styles/polymer-ha-style";
+import { HomeAssistant } from "../../types";
+
+@customElement("ha-long-lived-access-tokens-card")
+class HaLongLivedTokens extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
```
Comment (zsarnett) [inline src/panels/profile/ha-long-lived-access-tokens-card.ts:28]:
```suggestion
  @property({ attribute: false }) public hass!: HomeAssistant;
  
```

File: src/panels/profile/ha-long-lived-access-tokens-card.ts [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+            href="https://developers.home-assistant.io/docs/en/auth_api.html#making-authenticated-requests"
+            target="_blank"
+            rel="noreferrer"
+          >
+            ${this.hass.localize(
+              "ui.panel.profile.long_lived_access_tokens.learn_auth_requests"
+            )}
+          </a>
+          ${!accessTokens?.length
+            ? html` <p>
```
Comment (zsarnett) [inline src/panels/profile/ha-long-lived-access-tokens-card.ts:57]:
```suggestion
            ? html`<p>
```

File: src/panels/profile/ha-long-lived-access-tokens-card.ts [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+              "ui.panel.profile.long_lived_access_tokens.learn_auth_requests"
+            )}
+          </a>
+          ${!accessTokens?.length
+            ? html`<p>
+                ${this.hass.localize(
+                  "ui.panel.profile.long_lived_access_tokens.empty_state"
+                )}
+              </p>`
+            : accessTokens?.map(
```
Comment (bramkragten) [inline src/panels/profile/ha-long-lived-access-tokens-card.ts:63]:
```suggestion
            : accessTokens!.map(
```

File: src/panels/profile/ha-long-lived-access-tokens-card.ts [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+              "ui.panel.profile.long_lived_access_tokens.create"
+            )}
+          </mwc-button>
+        </div>
+      </ha-card>
+    `;
+  }
+
+  private async _createToken(): Promise<void> {
+    const name = await showPromptDialog(this, {
```
Comment (bramkragten) [inline src/panels/profile/ha-long-lived-access-tokens-card.ts:97]: No label for the input?

File: src/panels/profile/ha-long-lived-access-tokens-card.ts [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+    }
+
+    try {
+      const token = await this.hass.callWS<string>({
+        type: "auth/long_lived_access_token",
+        lifespan: 3650,
+        client_name: name,
+      });
+
+      setTimeout(() => {
```
Comment (bramkragten) [inline src/panels/profile/ha-long-lived-access-tokens-card.ts:114]: Why the timeout?

Comment (ludeeus) [reply src/panels/profile/ha-long-lived-access-tokens-card.ts:114]: Because the "close" event from the previous dialog kept force closing this one...

Comment (bramkragten) [reply src/panels/profile/ha-long-lived-access-tokens-card.ts:114]: Ah yeah 😄 that is annoying... But dont use setTimeout, uss `await new Promise((resolve) => setTimeout(resolve, 0));` (we should make a helper for that btw)

Comment (ludeeus) [reply src/panels/profile/ha-long-lived-access-tokens-card.ts]: Or just revert the previous screwup...

File: src/panels/profile/ha-long-lived-access-tokens-card.ts [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+        type: "auth/delete_refresh_token",
+        refresh_token_id: token.id,
+      });
+      fireEvent(this, "hass-refresh-tokens");
+    } catch (err) {
+      await showAlertDialog(this, {
+        title: this.hass.localize(
+          "ui.panel.profile.long_lived_access_tokens.delete_failed"
+        ),
+        text: err,
```
Comment (bramkragten) [inline src/panels/profile/ha-long-lived-access-tokens-card.ts:166]:
```suggestion
        text: err.message,
```

**Review by bramkragten — APPROVED** [review]

**Pull request merged**